src/ScheduledTasks/Client/ScheduleClientImpl.cs

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Represents a handle to a scheduled task, providing operations for managing the schedule.
+/// </summary>
+class ScheduleClientImpl : ScheduleClient
+{
+    readonly DurableTaskClient durableTaskClient;
+    readonly ILogger logger;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ScheduleClientImpl"/> class.
+    /// </summary>
+    /// <param name="client">The durable task client.</param>
+    /// <param name="scheduleId">The ID of the schedule.</param>
+    /// <param name="logger">The logger.</param>
+    /// <exception cref="ArgumentNullException">Thrown if <paramref name="client"/> or <paramref name="scheduleId"/> is null.</exception>
+    public ScheduleClientImpl(DurableTaskClient client, string scheduleId, ILogger logger)
+        : base(scheduleId)
+    {
+        this.durableTaskClient = client ?? throw new ArgumentNullException(nameof(client));
+        this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        this.EntityId = new EntityInstanceId(nameof(Schedule), this.ScheduleId);
+    }
+
+    /// <summary>
+    /// Gets the entity ID of the schedule.
+    /// </summary>
+    EntityInstanceId EntityId { get; }
+
+    /// <inheritdoc/>
+    public override async Task CreateAsync(ScheduleCreationOptions creationOptions, CancellationToken cancellation = default)
```

Comment:
We should not have duplicate implementations of `CreateAsync` or `DescribeAsync`. It is bad for code maintenance as subtle differences can sneak in. They are already not identical (one of them creates a log, the other one doesn't) which is almost certainly going to confuse us at some point. 

I would just have one of these call the other ones, instead of duplicating the code.